epilogue-processor/src/main/java/edu/wpi/first/epilogue/processor/EpilogueGenerator.java

```diff
@@ -152,13 +156,17 @@ public static boolean shouldLog(Logged.Importance importance) {
             out.println("  public static void bind(" + robotClassName + " robot) {");
             out.println("    robot.addPeriodic(() -> {");
             out.println("      long start = System.nanoTime();");
+            out.println("      config.loggingPeriod = Milliseconds.of(robot.getPeriod());");
+            out.println(
+                "      config.loggingPeriodOffset = Milliseconds.of(robot.getPeriod() / 2);");
```

Comment:
```suggestion
            out.println("      config.loggingPeriod = Seconds.of(robot.getPeriod());");
            out.println(
                "      config.loggingPeriodOffset = Seconds.of(robot.getPeriod() / 2);");
```
(You should also be able to remove the milliseconds import)
[`IterativeRobotBase.getPeriod()`](https://github.com/wpilibsuite/allwpilib/blob/ad45dc89efea734cc1952a290106e114ffbe6bf3/wpilibj/src/main/java/edu/wpi/first/wpilibj/IterativeRobotBase.java#L294) seems to be in seconds, since it returns `m_period`, which is initialized in [the constructor](https://github.com/wpilibsuite/allwpilib/blob/ad45dc89efea734cc1952a290106e114ffbe6bf3/wpilibj/src/main/java/edu/wpi/first/wpilibj/IterativeRobotBase.java#L83) from `period`, which the doc comment says is in seconds.

Reply:
Whoops :woman_facepalming: